docs/data/joy/customization/using-css-variables/using-css-variables.md

```diff
@@ -1,81 +1,97 @@
 # Using CSS variables
 
-<p class="description">Learn how to use CSS variables to customize Joy UI's components.</p>
+<p class="description">Learn how to use CSS variables to customize Joy UI components.</p>
 
-## Theme object
+## Introduction
 
-The `CssVarsProvider` reads the theme input (or the default theme) and creates the CSS variables according to the theme structure.
-It also creates an object that refers to the generated CSS variables under `theme.vars` so that you can use them from the JavaScript theme object.
+CSS variables are created by wrapping your app with the CssVarsProvider and using the `theme.vars` notation, allowing for an efficient customization experience.
```

Comment:
We could elaborate more here as well, but maybe this text is better suited to the "Usage" section, and should be replaced with the "what" and the "why" of CSS vars in the introduction

```suggestion
To use CSS variables, you must wrap your app with the `<CSSVarsProvider />` utility.
Then you can apply styles based on CSS variables using the `theme.vars.*` notation.
This notation is available to all styling APIs that Joy UI supports, including the `styled()` function and the `sx` prop.
```

Reply:
Reshuffled things around a bit — let me know what you think!